benchmarks/orc/generate_test_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "creating 100MB test file"
```

Comment:
this generates csv files, does it? should maybe orc conversion be invoked somewhere first? Else, let's remove the file.